elfi/env.py

```diff
@@ -1,12 +1,15 @@
 from collections import defaultdict
 import socket
+import logging
 
 from distributed import Client, LocalCluster
 from elfi.inference_task import InferenceTask
 
 _globals = defaultdict(lambda: None)
 _whitelist = ["client", "inference_task"]
 
+logging.getLogger('tornado').setLevel(logging.WARNING)
```

Comment:
I'd move this above the `_globals` definition. Also I think `logging.DEBUG` is a more appropriate logging level as warnings are actually often important to note.